api/HTMLCanvasElement.json

```diff
@@ -52,159 +52,6 @@
           "deprecated": false
         }
       },
-      "height": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLCanvasElement/height",
-          "support": {
-            "chrome": {
-              "version_added": "4"
```

Comment:
Chrome supports both this and width. Why is this being removed?

Reply:
There was a merge conflict regarding these lines due to #3474. The commit moves them to lines 492-542 (height) and 1071-1120. Not to worry, all of the data is still in this file. 😉